### PR TITLE
Pato wa py

### PR DESCRIPTION
**Pull request by PattoPeraltta**

### CODE REVIEW

File: refactor/messager/messager.py
Comment (unfoquito) [inline refactor/messager/messager.py:1]: This is ok! Now we use the method reply, which doesn't need the number of the user.

In production, would be ok to use send_text, but in testing mode, the WhatsApp API won't let us do it.

File: refactor/messager/messages.py
Comment (unfoquito) [inline refactor/messager/messages.py:1]: This is ok! Just added more messages.

File: refactor/toxicity/Analyzer.py
Comment (unfoquito) [inline refactor/toxicity/Analyzer.py:1]: This is not up to date. Just commited the latest version.

File: refactor/utilities/Fetcher.py
Comment (unfoquito) [inline refactor/utilities/Fetcher.py:1]: Ok. Just edited some comments.

File: refactor/whatsapp/wa.py
Comment (unfoquito) [inline refactor/whatsapp/wa.py:1]: Nice!

**Review by luzalbaposse — APPROVED** [review]
There were some conflicts between this branch and the v2.0-telegram, which was merged into v2.0 branch. 

Already solved, everything should work correctly : )

**Pull request merged**